CSS/CSS.sublime-syntax

```diff
@@ -140,11 +144,13 @@ contexts:
         2: punctuation.definition.keyword.css
       push:
         - meta_scope: meta.at-rule.import.css
-        - match: '\s*((?=;|\}))'
+        - match: '\s*(;|\})'
```

Comment:
I don't think a `}` should be scoped as `punctuation.terminator.rule.css`. I think this and the previous lookahead removed on line 45 (old) should probably remain as lookaheads. That way we don't need to duplicate the scope names for `;` and `}` in multiple places.